### PR TITLE
Remove sentry tracking and add popylar tracking.

### PR DESCRIPTION
**Pull request by arokem**

### CODE REVIEW

File: dmriprep/cli/run.py [hunk 1]
```diff
@@ -301,26 +300,11 @@ def main():
...
 
-    # Sentry tracking
-    if not opts.notrack:
-        from ..utils.sentry import start_ping
-        start_ping(run_uuid, len(subject_list))
-
     errno = 1  # Default is error exit unless otherwise set
     try:
         dmriprep_wf.run(**plugin_settings)
     except Exception as e:
```
Comment (oesteban) [inline dmriprep/cli/run.py]: The exception variable is not ever used : https://travis-ci.org/nipreps/dmriprep/jobs/636648631#L474

Comment (arokem) [reply dmriprep/cli/run.py]: Thanks. I believe I reinstated the logic in 85f0401

**Comment by arokem** [comment]
Now also rebased on master

File: dmriprep/__about__.py [hunk 2]
```diff
@@ -14,3 +14,5 @@
 DOWNLOAD_URL = (
     'https://github.com/nipreps/{name}/archive/{ver}.tar.gz'.format(
         name=__packagename__, ver=__version__))
+
+__ga_id__ = "UA-156165436-1"
```
Comment (oesteban) [inline dmriprep/__about__.py]: I'd keep this here, but in the long term, I'd like to think of a better config handling. Some kind of singleton pattern on a conf(ig) module.

**Pull request merged**